aura/lib/Aura/Languages.hs

```diff
@@ -224,6 +231,7 @@ buildPackages_1 (bt . pnName -> p) = \case
     Chinese    -> p <> " 正在构建中..."
     Esperanto  -> "Muntanta " <> p <> "..."
     Dutch      -> "Pakket " <> p <> " aan het bouwen..."
+    Ukrainian  -> "Збираємо " <> p <> "..."
```

Comment:
By the way, why three dots instead of one "…" here?  @fosskers It is just to mimic what pacman does in terminal? If so, it's a valid choice, but otherwise unicode three-dots in terminal with monospace font looks alright too.

Reply:
Let's review it again during the Localisation overhaul in Aura 4.